### ISSUE TITLE
Visual artifacts when viewing chests/shulker boxes.

### ISSUE THREAD

**Issue by Kichura** [FoundationGames/EnhancedBlockEntities#57]
When using enhanced block entities with sodium 0.3.x but without indium, Chests and shulker boxes will have a duplicated block which might be obvious to tell.

However with indium regardless, visual artifacts can be noticed when viewing the entities very closely but duplication is resolved from that.

A chest without indium mod present:
![image](https://user-images.githubusercontent.com/68134602/141354486-e38b7f54-6be0-401e-bf83-86bb11f5a360.png)

A shulker box with a transparent line present regardless of having indium or not:
![image](https://user-images.githubusercontent.com/68134602/141354531-07792bd5-50f7-4cf5-8fcb-68360bfda104.png)

and finally; Cancelling the chest animation causes it to flicker on specific frames which is absent in vanilla.

**Comment by Binero** [comment]
Sodium 0.4 with Indium installed:

![image](https://user-images.githubusercontent.com/6867251/145643439-84fc93ff-1a5d-49bf-a0cb-5e475704d1e6.png)

**Comment by TAKfsg** [comment]
Yes I am having he same problem. The chest flickers when you close it.
